packages/server/src/api/notifRoutes.ts

```diff
@@ -0,0 +1,112 @@
+import { ServerRoute, ResponseObject } from "@hapi/hapi";
+import { NotifBody } from "@template/common";
+import { NotifModel } from "../entity/NotifModel";
+import { NotifPayload } from "../joi";
+import * as dotenv from "dotenv";
+import * as webPush from "web-push";
+import { DeepPartial } from "typeorm";
+
+// configure env vars for VAPID
+dotenv.config();
+
+// if env vars not found, then throw an error early
+if (!process.env.EMAIL || !process.env.PUBLICKEY || !process.env.PRIVATEKEY) {
+  throw new Error(
+    "please add a .env file with keys+email in packages/server. ask alex/eddy for deets."
```

Comment:
can you add a `.env.example` with the env without values

Reply:
also what do u think about writing a script to generate it, and then binding it to a yarn command?

Reply:
could also do `.env.development` with values filled in but the values are all just placeholder

Reply:
then when you do `dotenv.config` you can specify the path of the dotenv file and switch it based on `process.env.NODE_ENV`

Reply:
actually we did it here so y'all don't need to https://github.com/sandboxnu/office-hours/commit/4ca8f8a7b65caf0a0c8596fa04d6a9081952016d

Reply:
I think we're going to TODO this one until something's merged